src/components/Base/Button/index.jsx

```diff
@@ -1,34 +1,52 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import isFunction from 'lodash/isFunction';
 
-import { Button } from 'reactstrap';
 import styles from './index.scss';
 
-export default class PiButton extends PureComponent {
+export default class Button extends PureComponent {
   static propTypes = {
-    active: PropTypes.bool,
-    block: PropTypes.bool,
-    color: PropTypes.string,
+    type: PropTypes.string,
+    className: PropTypes.string,
+    style: PropTypes.object,
+    loading: PropTypes.bool,
     disabled: PropTypes.bool,
-    tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
-    onClick: PropTypes.func,    
+    onClick: PropTypes.func,
+    isSubmit: PropTypes.bool,        
   };
 
   static defaultProps = {
-    color: 'secondary'
+    type: 'default',
   };  
 
+  handleClick = (e) => {
+    let isDisabled = this.props.disabled;
```

Comment:
use 'const' instead of 'let' if variable won't be modified

Reply:
done